brave-intl/publishers
# Implement a new HtmlValidator

Create a new HtmlValidator. HtmlValidator can be included in any
ActiveRecord model with the `valitades_with` directive.

When no `:attributes` options are passed all the model attributes
are automatically validated against his HTML entities blocklist.
When `:attributes` is passed only the selected model attributes are
validated against the blocklist.

The HtmlValidator has also an allowlist that can be enabled on a
attribute by attribute basis, by adding the `html: true` option to
each validates call, specifying an attribute.

## Pull Request Name

Implement a new HtmlValidator

#### Features

Implement a new HtmlValidator to fix issue #2828 

#### How To Test

1. Setup a rails console
2. Create a new publisher with name containing an HTML tag, it should be rejected with an error
3. All the rest should work the same

#### Pull Request Checklist
